specs/ReleaseChannelPreferenceSet.md

```diff
@@ -0,0 +1,325 @@
+issues before they reach the stable WebView2 Runtime and affect end users.
+
+## Channel overview
+
+|Channel|Primary purpose|How often updated with new features|
+|:---:|---|:---:|
+|Stable (WebView2 Runtime)|Broad Deployment|~4 weeks|
+|Beta|Flighting with inner rings, automated testing|~4 weeks|
+|Dev|Automated testing, selfhosting to test new APIs and features|Weekly|
+|Canary|Automated testing, selfhosting to test new APIs and features|Daily|
+
+Currently you can test against prerelease channels by setting the `RuntimeChannelPreference` or
+`BrowserExecutableFolder` overrides, described in [Test upcoming APIs and features](https://learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/set-preview-channel).
+These overrides are useful for testing experimental APIs but are not designed to support more
+complex automated testing and flighting scenarios.
+
+To make it easier to implement prerelease testing, the WebView2 team is adding new environment
+options for channel selection, `RuntimeChannelPreferenceSet` and `ShouldReverseChannelSearchOrder`.
+These options can be used together to specify which channels the WebView2 loader should search for
+in order of least to most stable: Canary -> Dev -> Beta -> Stable WebView2 Runtime.
+For example, you can set `ShouldReverseChannelSearchOrder` to true and specify {Beta, Stable}
+to tell the WebView2 loader to try running the app with Beta channel if it exists on the machine,
+and otherwise fall back on the stable WebView2 Runtime. With the new options you can:
+- Run automated tests against different channels on a machine with multiple channels installed
+- Target a particular channel when flighting
+- Apply different channels to different flighting rings
+- Fall back on a more stable preview channel
+
+# Examples
+## RuntimeChannelPreferenceSet
+The runtime channel preference set and channel search order are specified on the
+CoreWebView2EnvironmentOptions prior to creating the WebView2 environment.
+```c#
+CoreWebView2Environment _webViewEnvironment;
+async System.Threading.Tasks.Task CreateEnvironmentAsync() {
+    CoreWebView2RuntimeChannel channels =
+        CoreWebView2RuntimeChannel.Beta | CoreWebView2RuntimeChannel.Stable;
+    CoreWebView2EnvironmentOptions customOptions = new CoreWebView2EnvironmentOptions
+    (
+        shouldReverseChannelSearchOrder: true,
+        runtimeChannelPreferenceSet: channels
+    );
+    // Use GetAvailableCoreWebView2BrowserVersionStringWithOptions to check which
+    // channel is used with the custom options. If Beta channel was not found,
+    // install it on the device.
+    string version =
+        CoreWebView2Environment.GetAvailableBrowserVersionStringWithOptions(
+            browserExecutableFolder: null,
+            environmentOptions: customOptions);
+    if (!IsBetaChannel(version))
+    {
+        InstallChannel(CoreWebView2RuntimeChannel.Beta);
+    }
```

Comment:
I thought it would be good to include as a fallback in case something happens, like Beta installation fails (though I see how it's not really used in the example)

Reply:
Please add a comment explaining or update sample to actually require more than one channel to show this off.